openforcefields/tests/test_water_models.py

```diff
@@ -91,6 +91,34 @@ def test_tip3p_fb(water_molecule):
         },
     )
 
+# TODO: The interchange that results from applying the OPC water model yields a sigma that is exactly
+#  twice what it should be on the hydrogens. This problem is only cosmetic as the epsilon is 0.
```

Comment:
Any suspicion if this is a problem with the force field or Interchange?

Reply:
We squinted a lot at the force field and couldn't spot a problem there, but it's a very confusing issue. The things we figured out are:

* The atom/particle indexing are the same between the OpenMM-generated system and the OpenFF-generated one
* The disagreeing sigmas are `0.890898` and `1.78179743628`, which are `1 / (2^(1/6))` and `2 / (2^(1/6))` respectively. 

We didn't see any mention of `rmin_half=2. * unit.angstrom`, which would seem to be the only way we should end up with the latter. So it seems like it may be an interchange bug, maybe having to do with the multiplicity of matches in the vsites. @lilyminium is working on a minimal reproducing example and will open an issue when she has one.